bundles/org.openhab.binding.smartthings/README.md

```diff
@@ -124,7 +124,7 @@ Here is a sample configuration for a RGB bulb, such as a Sengled model E11-N1EA
 
 ## References
 
-1. [openHAB configuration documentation](http://docs.openhab.org/configuration/index.html)
+1. [openHAB configuration documentation](http://openhab.org/docs/configuration/index.html)
```

Comment:
Thanks. What about changing the protocol to HTTPS?

Reply:
What about ignoring, that i simply didnt see this and wait for my next commit. 🙈